src/controllers/userController.ts

```diff
@@ -1,38 +1,67 @@
-import { Request, Response } from 'express';
-import { validationResult } from 'express-validator';
-import { registerUser } from '../services/userService';
-import { verifyPassword } from '../services/userService';
```

Comment:
added these two lines together only,  I didn't change the rest of the file, it's only automatic formatting.